docs/prerequisites.adoc

```diff
@@ -90,27 +80,29 @@ Follow the steps below to configure your path on Windows:
 . Close all open windows by clicking 'OK'
 . Open a new terminal and verify terraform has been properly installed
 
-==== Testing Terraform installation
+=== Testing Terraform installation
 
-Open a terminal and test:
+Open a terminal type this command:
 
 [source,bash]
 ----
-terraform -v
-Terraform v0.12.24
+> terraform -v
```

Comment:
Always provide the expected/sample output, i.e.

```shell
$ terraform -v
Terraform v0.14.7
```

Reply:
I think I will remove this section:
- that's part of the installation process
- output being a version, it will quickly become obsolete and ask for further maintenance :-)